debug/src/debug.js

```diff
@@ -1,11 +1,29 @@
 import { checkPropTypes } from 'prop-types';
 import { getDisplayName } from './devtools/custom';
 import { options, toChildArray } from 'preact';
+import { ELEMENT_NODE, TEXT_NODE, DOCUMENT_NODE } from './constants';
 
 export function initDebug() {
 	/* eslint-disable no-console */
 	let oldBeforeDiff = options.diff;
 
+	options.root = (vnode, parentNode) => {
+		if (!parentNode) {
+			throw new Error('Undefined parent passed to render(), this is the second argument.\nCheck if the element is available in the DOM/has the correct id.');
+		}
+		let isValid;
+		switch (parentNode.nodeType) {
+			case ELEMENT_NODE:
+			case DOCUMENT_NODE: isValid = true; break;
+			default: isValid = false;
+		}
+		if (!isValid) throw new Error(`
+			Expected a valid HTML node as a second argument to render.
+			Received VALUE instead: render(<${vnode.type.name || vnode.type} />, VALUE);
```

Comment:
We could replace `VALUE` with the actual value of the passed `parentNode` and cast it to a string :+1:

Reply:
Like this typeof parentNode === 'object' ? JSON.stringify(parentNode) : parentNode?

Reply:
Just inserting `parentNode` should be fine as it would be automatically cast to a string. It may even be advantageous for us if it's just `[object Object]` as that looks really wrong and like something must be up :D

Reply:
Allright, fixed it :)